public/locales/pt-BR/common.json

```diff
@@ -0,0 +1,427 @@
+  "Add or update feed from Podcast Index": "Adicione ou atualize feed do Podcast Index",
+  "Add to Playlist": "Adicionar a Playlist",
+  "Admin Tools": "Ferramentas Administrativas",
+  "Anonymous": "Anônimo",
+  "Are you sure you want to delete this clip": "Apagar clipe?",
+  "Are you sure you want to delete this playlist": "Apagar playlist?",
+  "Are you sure you want to remove all of your history items": "Apagar todo histórico?",
+  "Are you sure you want to remove all of your queue items": "Remover fila?",
+  "ARIA - Toggle UI theme change": "Alteração do tema da Interface (UI)",
+  "ARIA - Toggle video only mode": "Apenas vídeo",
+  "ARIA – Dropdown helper description": "Pressione a tecla Enter ou Seta para Baixo para exibir o menu suspenso. Então use a tecla TAB, ou seta para cima ou para baixo para focar nos itens disponíveis.",
+  "Auto Queue": "Fila automática",
+  "Auto-scroll button - aria description": "",
+  "Auto-scroll button - off": "Rolagem automática: Desligado",
+  "Auto-scroll button - on": "Rolagem automática: Ligada",
+  "Below are a few ways you can support the project": "Abaixo estão algumas formas de apoiar o projeto.",
+  "Below is a table comparing free and premium features": "Veja abaixo uma tabela comparando os recursos gratuitos e premium.",
+  "brandName chat is powered by": "O bate-papo ao vivo do Podverse é movido por essas bibliotecas de código aberto",
+  "Bug Report": "Reportar um erro",
+  "Buy a Podverse premium membership": "Assine o Podverse Premium",
+  "Cancel": "Cancelar",
+  "category - arts": "Artes",
+  "category - business": "Negócios",
+  "category - comedy": "Comédia",
+  "category - education": "Educação",
+  "category - fiction": "Ficcção",
+  "category - government": "Governo",
+  "category - healthfitness": "Saúde & Exercícios Físicos",
+  "category - history": "História",
+  "category - kidsfamily": "Crianças & Família",
+  "category - leisure": "Lazer",
+  "category - music": "Música",
+  "category - news": "Notícias",
+  "category - religionspirituality": "Religião & Espiritualidade",
+  "category - science": "Ciência",
+  "category - societyculture": "Sociedade & Cultura",
+  "category - sports": "Esportes",
+  "category - technology": "Tecnologia",
+  "category - truecrime": "Crimes",
+  "category - tvfilm": "TV & Filmes",
+  "Chapters": "Capítulos",
+  "Claim free premium membership": "Para reivindicar sua assinatura premium grátis, entre na sua conta do Podverse e pressione o botão Reivindicar Token abaixo.",
+  "Clip": "Clipe",
+  "Clip times": "Duração do clipe",
+  "Clip title": "Título do clipe",
+  "Clips": "Clipes",
+  "Clips from this episode": "Clipes deste episódio",
+  "Close": "Fechar",
+  "Comments": "Comentários",
+  "Confirm Password": "Confirme a Senha",
+  "ConfirmDeleteModalLine1": "Excluir sua conta para sempre?",
+  "ConfirmDeleteModalLine2": "Escreva a palavra \"delete\" para apagar sua conta, então pressione \"Apagar Conta\".",
```

Comment:
Here I kept `delete` in English because I guessed that we are not dealing with translations of this word on JS verification. Is that correct?

Reply:
Same applies for key `Type delete`.

Reply:
Good catch. Yes it's hardcoded to English here...maybe we should change that, but I think it's ok to retain the English hardcoded handling for now.

https://github.com/podverse/podverse-web/blob/develop/src/components/Modals/Auth/ConfirmDeleteAccountModal.tsx#L23